File: trash/Python/Nalison Python/Python I/IMC.py
"""
A organização mundial de saúde adotou algumas regras para o cálculo da
massa corporal, definindo o peso ideal. Para que o cálculo seja realizado de
maneira correta os seguintes dados devem ser informados: peso e altura. A
seguir, tabela base para o desenvolvimento de um algoritmo capaz de calcular
o IMC conforme dados informados(peso e altura):
"""
nome = str(input("Digite seu nome: "))
peso = float(input("Seu peso: "))
altura = float(input("Sua altura: "))
imc = peso/(altura*altura)

print("Seu imc é: ",imc,)

if imc < 18.5:
    print("Portanto é menor que 18,5: Abaixo do peso")
elif imc <= 18.5 and 24.9:
    print("Portanto está entre 18,5 e 24,9: Peso normal")
elif imc <= 29.9:
    print("Portanto está entre 25,0 e 29,9: Sobrepeso")
elif imc <= 39.9:
    print("Portanto está entre 30,0 e 39,9: Obesidade")
else:
    print("Portanto está maior que 40,0: Obesidade Grave")

File: HTML/imc.js
function calcularIMC() {
    let peso = parseFloat(document.getElementById("peso").value);
    let altura = parseFloat(document.getElementById("altura").value);

    let imc = peso / (altura * altura);
    let classificacao = "";

    if (imc < 18.5) {
    classificacao = "MAGRISSIMO";
    } else if (imc < 25) {
    classificacao = "Peso normal";
    } else if (imc < 30) {
    classificacao = "Sobrepeso";
    } else if (imc < 35) {
    classificacao = "Obesidade grau 1";
    } else if (imc < 40) {
    classificacao = "Obesidade grau 2";
    } else {
    classificacao = "Obesidade grau 3";
    }

    document.getElementById("resultado").innerText = 
    "Seu IMC é " + imc.toFixed(2) + " (" + classificacao + ")";
}